app-backup/burp/files/burp-2.4.0-fix-musl-strptime.patch

```diff
@@ -0,0 +1,82 @@
+From 76b7f1ba9f4445108059b13f3d79a7fde8a292a3 Mon Sep 17 00:00:00 2001
```

Comment:
I'm going to throw in a link to the bug & commit: https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto.